imap_processing/codice/codice_l1a.py

```diff
@@ -243,20 +291,18 @@ def unpack_science_data(self, packets: list):
         ----------
         packet : list[space_packet_parser.parser.Packet]
             List of packets for the APID of interest
-
-        TODO: Check to see if we expect to have multiple packets?
         """
         self.compression_algorithm = LO_COMPRESSION_ID_LOOKUP[self.view_id]
         self.collapse_table_id = LO_COLLAPSE_TABLE_ID_LOOKUP[self.view_id]
 
         science_values = packets[0].data["DATA"].raw_value
 
         # Divide up the data by the number of priorities or species
-        num_bytes = len(science_values)
+        num_bits = len(science_values)
         chunk_size = len(science_values) // self.num_counters
 
         self.data = [
-            science_values[i : i + chunk_size] for i in range(0, num_bytes, chunk_size)
+            science_values[i : i + chunk_size] for i in range(0, num_bits, chunk_size)
```

Comment:
Do you know if these will be integers and could convert them right away?

```suggestion
            int(science_values[i : i + chunk_size], 2) for i in range(0, num_bits, chunk_size)
```

Reply:
Not quite, these are intended to be a list of 128 bits